### ISSUE TITLE
Port is not right in quickstart

### ISSUE THREAD

**Issue by krancour** [Azure/open-service-broker-azure#125]
The following line from the quickstart is meant to open Wordpress, but doesn't universally work because the node port the service is assigned to will not always bee `31215`.

```
open http://$(minikube ip):31215/admin
```

**Comment by arschles** [comment]
would `open http://$(minikube service wordpress-wordpress-service --format {{.IP}}):80` work?

**Comment by krancour** [comment]
> would open http://$(minikube service wordpress-wordpress-service --format {{.IP}}):80 work?

Assuming that the `helm install` command specifies no namespace (as the instructions currently are), the "correct" command would be this:

```
$ minikube service quickstart-wordpress"
```

But there's some weirdness here...

First, see what the help says for this command:

```
```

**Comment by krancour** [comment]
> would open http://$(minikube service wordpress-wordpress-service --format {{.IP}}):80 work?

Assuming that the `helm install` command specifies no namespace (as the instructions currently are), the "correct" command would be this:

```
$ minikube service quickstart-wordpress"
```

But there's some weirdness here...

First, see what the help says for this command:

```
$ minikube service -h
Gets the kubernetes URL(s) for the specified service in your local cluster. In the case of multiple URLs they will be printed one at a time.
```

It's only supposed to _list_ the URLs, but what it _really_ does is _opens_ each in the default browser. (I'm going to open an issue against minikube having to do with this discrepancy.)

Second issue, the command will open _all_ of the URLs in the default browser. This would include _both_ the URL that involves the node port mapped to port 80 _and_ the URL that involves the node port mapped to port 443., but since the default format is `"http://{{.IP}}:{{.Port}}"`, for one of these two, you'll end up seeing this in your browser:

```
Bad Request

Your browser sent a request that this server could not understand.
Reason: You're speaking plain HTTP to an SSL-enabled server port.
Instead use the HTTPS scheme to access this URL, please.
```

That's no good.

Bottom line... we cannot rely on `minikube service ...`

My kubectl-query-fu is lacking, but is there a way to tease the right URL out of a `kubectl` command?

**Comment by krancour** [comment]
I overlooked the `--url` option...

I still think this might not be optimal (feel free to disagree, please):

```
$ mk service quickstart-wordpress --url
http://192.168.99.100:30080
http://192.168.99.100:31126
```

^ Which one is http/80 and which one is https/443? It's not obvious. The directions would have to say something like "try both of these and one of them will work." That doesn't feel completely satisfactory.

**Comment by krancour** [comment]
Ok... so this is almost moot...

Since #128 switches the quickstart ti Ghost and Ghost only exposes port 80.

**Comment by carolynvs** [comment]
Yes, I'm sorry that I wimped out on figuring out how to get the URL nicely. Everything you posted here is what I ran into as well, and I don't know why I thought the port was safe to pick... 

Can we just close this and not worry about it since we are going to use ghost?

**Comment by krancour** [comment]
> Can we just close this and not worry about it since we are going to use ghost?

Still can't hardcode it, but `minikube service quickstart-wordpress --url` should be ok... but someone besides me verify that.

**Comment by krancour** [comment]
And now we're not using Ghost because that chart seems to have its own issues with mk. Not sure yet about other distros.

**Comment by jeremyrickard** [comment]
I think I solved this with JSON Path!

```console
open http://$(minikube ip):$(kubectl get service quickstart-wordpress -o jsonpath={.spec.ports[?\(@.name==\"http\"\)].nodePort})
```
